clang/lib/3C/ArrayBoundsInferenceConsumer.cpp

```diff
@@ -15,6 +15,7 @@
 #include "clang/3C/ConstraintResolver.h"
 #include "clang/Analysis/Analyses/Dominators.h"
 #include "clang/Analysis/CFG.h"
+#include <cctype>
```

Comment:
This is to solve one of the Windows-only bugs?

Reply:
Yes. That's part of the merge commit, so it doesn't  show up in the overall diff.